packages/adapter-unstorage/tests/redis-json.test.ts

```diff
@@ -0,0 +1,103 @@
+import { createStorage } from "unstorage"
+import { runBasicTests } from "utils/adapter"
+import { hydrateDates, UnstorageAdapter } from "../src"
+import { defineDriver } from "unstorage"
+import { createClient } from "redis"
+import type { RedisClientType, RedisClientOptions } from "redis"
+
+globalThis.crypto ??= require("node:crypto").webcrypto
+
+const redisJSONDriver = defineDriver((options: RedisClientOptions) => {
```

Comment:
Would it be appropriate to add a test with a custom driver to this project?

Reply:
Yeah tbh i think it's a valid use-case, works for me!